packages/yourloops/test/integration/pages/patient-dashboard/patient-dashboard.hcp.spec.tsx

```diff
@@ -133,16 +139,28 @@ describe('Patient dashboard for HCP', () => {
       selectedTeamName
     }
 
+    const appMainLayoutParams: AppMainLayoutHcpParams = {
+      footerHasLanguageSelector: false,
+      headerInfo: {
+        loggedInUserFullName: `${firstName} ${lastName}`,
+        teamMenuInfo: {
+          selectedTeamName,
+          isSelectedTeamPrivate: false,
+          availableTeams: buildAvailableTeams()
+        }
+      }
+    }
+
     await act(async () => {
       renderPage(patientDashboardRoute)
     })
 
+    await testAppMainLayoutForHcp(appMainLayoutParams)
```

Comment:
```suggestion
```
I don't know why you added this here. It is already tested in `should render correct components when navigating to a patient not scoped on the private team` and `should render correct components when navigating to a patient scoped on the private team`